sources/packages/backend/apps/db-migrations/src/sql/StudentLoanBalances/Create-student-loan-balances.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE sims.student_loan_balances(
+  id SERIAL PRIMARY KEY,
+  student_id INT NOT NULL REFERENCES sims.students(id),
+  csl_balance NUMERIC(8, 2) NOT NULL,
+  balance_date TIMESTAMP WITH TIME ZONE,
+  -- Audit columns
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
+  updated_at timestamp without time zone NOT NULL DEFAULT now(),
+  creator INT NULL DEFAULT NULL REFERENCES users(id) ON DELETE
```

Comment:
I believe there was an agreement to stop using the `ON DELETE`. I was trying to check the latest tables added.
I believe that the latest decision was to them be added as below.
```sql
created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
creator INT NULL DEFAULT NULL REFERENCES sims.users(id),
modifier INT NULL DEFAULT NULL REFERENCES sims.users(id)
```